README.md

```diff
@@ -8,3 +8,16 @@ You'll need the version of python in the .python-version file. I suggest using p
 2. `pyenv install 3.10.1`
 3. `pip install -r requirements.txt`
 
+## Run the app
+
+1. `uvicorn main:app --reload`
+
+## Starting Docker
```

Comment:
We are for now, to get it running faster. With SQLAlchemy it should just be a drop in for any other database since SQLite is more limited than other versions. So if we wanted to use GIS or JSON datatypes we can add them when when we switch to PostgreSQL or MySQL.